app/models/service_template_transformation_plan.rb

```diff
@@ -24,6 +24,8 @@ def transformation_mapping_resource
   end
 
   def validate_order
+    # Service template should not be orderable if all VMs have already been migrated
+    return false if vm_resources.reject { |res| res.resource.is_tagged_with?('transformation_status/migrated', :ns => '/managed') }.empty?
```

Comment:
You could simplify by getting rid of the `return false if` and the `true` below. Otherwise, looks good.